pom.xml

```diff
@@ -466,15 +466,21 @@
 				<artifactId>mockito-core</artifactId>
 				<version>1.10.19</version>
 			</dependency>
+			<dependency>
+				<groupId>org.powermock</groupId>
```

Comment:
What happens when you do not add in this new dependency?

Reply:
@dkayiwa I still have a successful build locally

Reply:
So why did you add the dependency?

Reply:
All previous builds failed and adding the dependency caused less errors, let me update the PR then